### ISSUE TITLE
`operation_id_case_convention` rule can't be easily disabled

### ISSUE THREAD

**Issue by Richienb** [IBM/openapi-validator#312]
With every other case convention rule, it can be disabled like this:

```js
{
	param_name_case_convention: ["off"]
}
```

If the first item in the array is `"off"`, then the linter will skip the rule.

`operation_id_case_convention` is the only rule that does not follow this. Despite providing `"off"` as the first item, it still tries to check the case which isn't provided:

https://github.com/IBM/openapi-validator/blob/2bf13e19fd3893190caa0725671197d92a086464/src/plugins/validation/2and3/semantic-validators/operations-shared.js#L112-L114

The code should only run this check if `checkStatus !== "off"` like every other rule.

For now, a temporary workaround is to specify a case anyways which will disable the check:

```js
{
	operation_id_case_convention: ["off", "lower_snake_case"]
}
```

**Comment by dpopp07** [comment]
That does sound like unexpected behavior. I will try look into that soon but I'm also happy to review a PR if you would like to open one.
